### DISCUSSION TITLE
Github auth does not work for this git clone

### DISCUSSION THREAD

**Discussion by yytina** [community/community#22700] [New to GitHub]
<p><code>git clone https://github.com/Homebrew/homebrew-cask-cask /usr/local/Homebrew/Library/Taps/homebrew/homebrew-cask-cask</code><br>
…<br>
…<br>
Password for ‘<a href="https://yytina@github.com" rel="nofollow noopener">https://yytina@github.com</a>’:<br>
remote: Invalid username or password.<br>
fatal: Authentication failed for ‘<a href="https://github.com/Homebrew/homebrew-cask-cask/" rel="nofollow noopener">https://github.com/Homebrew/homebrew-cask-cask/</a>’</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>It looks as though the repository you’re trying to clone either doesn’t exist or is private. Either way, you don’t seem to have access to it.</p>
<p>If you’re using a tutorial, it could be that it’s out of date and you need to find something newer.</p>
